util/wrapper/stdlib.h

```diff
@@ -0,0 +1,19 @@
+#ifndef LIBUTIL_STDLIB_H
+#define LIBUTIL_STDLIB_H
+
+/* Include the real stdlib.h. We don't want to recreate all the stuff */
+/* it contains - just add libutil_isatty declaration to it */
+#include_next <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const char* getprogname(void) __asm__ ("libutil_getprogname");
+void setprogname(const char *) __asm__ ("libutil__setprogname");
```

Comment:
```suggestion
const char* libutil_getprogname(void);
void libutil_setprogname(const char *);
const char* getprogname(void) __asm__ ("libutil_getprogname");
void setprogname(const char *) __asm__ ("libutil__setprogname");
```

Now you've gotten rid of the libutil prototypes...

Reply:
Ohhhh makes more sense now. I thought it was just the last 2 lines and had no idea how it was making the alias.